Automattic/woocommerce-payments
# Clean up after order is completed processing

Fixes #7423

#### Changes proposed in this Pull Request
- Added the existing cleanup code in gateway into Processed state
- Clean up code includes
    - clearing cart
    - reducing stock levels
    - clearing UPE intents from session
- Fixed tests. 

#### Testing instructions

Checkout using the new payment process and ensure that cleanup code is called.

-------------------


**Post merge**
